### ISSUE TITLE
"aria-labelledby" references an invalid element if there is a single tab

### ISSUE THREAD

**Issue by danxuliu** [nextcloud/nextcloud-vue#3032]
The `aria-labelledby` attribute must point to an element in the same document. In the `AppSidebarTab` [it references its tab header](https://github.com/nextcloud/nextcloud-vue/blob/e499b4e9b018f8ef6a5d2a05b3d02a6670e5ce4b/src/components/AppSidebarTab/AppSidebarTab.vue#L30), but [when there is a single tab the tab headers are removed](https://github.com/nextcloud/nextcloud-vue/blob/e499b4e9b018f8ef6a5d2a05b3d02a6670e5ce4b/src/components/AppSidebar/AppSidebarTabs.vue#L30). Therefore, in that case [the `aria-labelledby` should be replaced by `aria-label` with the tab name](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby), which probably would require passing `hasMultipleTabs` (or something equivalent) to `AppSidebarTab` (which I do not know how to do :-) ) to set one or the other.
